controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java

```diff
@@ -553,6 +553,11 @@ public void lockOrThrow(ApplicationId applicationId, Consumer<LockedApplication>
     }
 
     public void notifyJobCompletion(JobReport report) {
+        log.log(Level.INFO, String.format("Notified of %s of %s %d for '%s'.",
```

Comment:
Is this needed? Can't say I ever missed it when debugging. Access log will also contain received notifications (although with fewer details).

Reply:
I missed it when debugging the controller, but perhaps I just didn't know where to look? 
I added it primarily for debugging some tests now, though, so I won't object to removing it again.

Reply:
Ok, I would just remove it or change level to `DEBUG`.